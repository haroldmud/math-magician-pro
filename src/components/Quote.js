import React, { useState } from 'react';
import Header from './Header';
import '../styles/Quote.css';

const Quote = () => {
  const [links] = useState([
    { link: '../Calculator', active: false },
    { link: '../', active: false },
    { link: '../Quote', active: true },
  ]);

  return (
    <section>
      <Header anchor={links} />
      <h3>
        Mathematics is not about numbers, equations, computations
        or algorithms: it's about understanding. -willian Paul Thurston
      </h3>
    </section>
  );
};

export default Quote;
