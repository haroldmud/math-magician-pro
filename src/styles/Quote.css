section h3 {
  text-align: center;
  margin-top: 7rem;
  font-size: 20px;
}
