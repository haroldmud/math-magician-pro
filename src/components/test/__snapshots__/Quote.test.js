import React from 'react';
import renderer from 'react-test-renderer';
import { HashRouter as Router } from 'react-router-dom';
import Quote from '../../Quote';
import '@testing-library/jest-dom';

describe('test of the home page', () => {
  it('test Quote rendering', () => {
    const quoteTest = renderer
      .create(
        <Router>
          <Quote />
        </Router>,
      )
      .toJSON();
    expect(quoteTest).toMatchSnapshot();
  });
});
