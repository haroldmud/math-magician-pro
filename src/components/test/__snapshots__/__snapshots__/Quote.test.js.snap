// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test of the home page test Quote rendering 1`] = `
<section>
  <header>
    <h1>
      Math Magician
    </h1>
    <nav>
      <a
        className="menu-item"
        href="#/"
        onClick={[Function]}
      >
        Home
      </a>
      <a
        className="menu-item"
        href="#/Calculator"
        onClick={[Function]}
      >
        Calculator
      </a>
      <a
        className="menu-item"
        href="#/Quote"
        onClick={[Function]}
      >
        Quote
      </a>
    </nav>
  </header>
  <h3>
    Mathematics is not about numbers, equations, computations or algorithms: it's about understanding. -willian Paul Thurston
  </h3>
</section>
`;
